src/core/_config.scss

```diff
@@ -183,6 +183,13 @@ $shadow-hover: 2px 8px 8px -5px $shadow-color-hover !default;
   transform: translate($x, $y);
 }
 
+@mixin transition($name:all, $duration:235ms, $animation:ease-in-out, $delay:0s){
```

Comment:
it seems that most other functions in this file have comments, could you add comments to this function?